dandi/metadata.py

```diff
@@ -260,6 +260,14 @@ def extract_wasAttributedTo(metadata):
         return [wat]
 
 
+def extract_wasGeneratedBy(metadata):
+    wat = extract_model(models.Session, metadata, identifier=metadata.get("session_id"))
```

Comment:
ideally this would be name, not identifier.

Reply:
suggest the change? ;)